ionoscloud/resource_server.go

```diff
@@ -1529,23 +1542,39 @@ func setResourceServerData(ctx context.Context, client *ionoscloud.APIClient, d
 		return fmt.Errorf("server entities cannot be empty for %s", d.Id())
 	}
 
-	if server.Properties != nil && server.Properties.BootVolume != nil {
-		volume, apiResponse, err := client.ServersApi.DatacentersServersVolumesFindById(ctx, datacenterId, d.Id(), *server.Properties.BootVolume.Id).Execute()
-		logApiRequestTime(apiResponse)
-		if err == nil {
-			var volumes []interface{}
+	inlineVolumeIds := d.Get("inline_volume_ids")
+
+	if inlineVolumeIds != nil {
+		inlineVolumeIds := inlineVolumeIds.([]any)
+
+		var volumes []any
+		for i, volumeId := range inlineVolumeIds {
+			volume, apiResponse, err := client.ServersApi.DatacentersServersVolumesFindById(ctx, datacenterId, d.Id(), volumeId.(string)).Execute()
+			logApiRequestTime(apiResponse)
+			if err != nil {
+				return fmt.Errorf("error retrieving inline volume %w", err)
+			}
+			volumePath := fmt.Sprintf("volume.%d.", i)
```

Comment:
@cbeti-ionos please add a comment about what we discussed earlier

Reply:
added a comment to remind that this portion will require more attention once multiple inline volumes are actually implemented